governance/src/agent.spec.js

```diff
@@ -0,0 +1,417 @@
+const {
+  Finding, createTransactionEvent, TransactionEvent, ethers, FindingType, FindingSeverity,
+} = require('forta-agent');
+
+const { provideHandleTransaction, provideInitialize } = require('./agent');
+const { getAbi } = require('./utils');
+const { createMockEventLogs, getObjectsFromAbi } = require('./test-utils');
+
+const config = require('../bot-config.json');
+
+const MINIMUM_EVENT_LIST = [
+  'ProposalCreated',
+  'ProposalCanceled',
+  'ProposalExecuted',
+];
```

Comment:
This isn't really needed for this Bot.  For the governance bot template that this is derived from, it made sense to check for some minimal support of events that were in common across many different governance contract ABIs.  Here we are only supported one specific contract ABI, so we don't need to do this check.
```suggestion
```